addons/website/static/src/js/tours/banner.js

```diff
@@ -1,106 +0,0 @@
-}, {
-    trigger: ".o_we_customize_panel",
-    extra_trigger: "#wrapwrap .s_cover h1:not(:containsExact(\"Catchy Headline\"))",
-    content: _t("Customize any block through this menu. Try to change the background color of this block."),
-    position: "right",
-}, {
-    trigger: '.o_we_add_snippet_btn',
-    content: _t("Go back to the blocks menu."),
-    position: 'bottom',
-}, {
-    trigger: "#snippet_structure .oe_snippet:eq(3) .oe_snippet_thumbnail",
-    content: _t("Drag another block in your page, below the cover."),
-    position: "bottom",
-    run: "drag_and_drop #wrap",
-}, {
-    trigger: "button[data-action=save]",
-    content: _t("Click the <b>Save</b> button."),
-    position: "bottom",
-}, {
-    trigger: "a[data-action=show-mobile-preview]",
-    content: _t("Good Job! You have designed your homepage. Let's check how this page looks like on <b>mobile devices</b>."),
-    position: "bottom",
-}, {
-    trigger: ".modal-dialog:has(#mobile-viewport) button[data-dismiss=modal]",
-    content: _t("After having checked how it looks on mobile, <b>close the preview</b>."),
-    position: "right",
-}, {
-    trigger: "#new-content-menu > a",
-    content: _t("<p><b>Your homepage is live.</b></p><p>Let's add a new page for your site.</p>"),
-    position: "bottom",
-},  {
-    trigger: "a[data-action=new_page]",
-    content: _t("<p><b>Click here</b> to create a new page.</p>"),
-    position: "bottom",
-}, {
-    trigger: ".modal-dialog #editor_new_page input[type=text]",
-    content: _t("<p>Enter a title for the page.</p>"),
-    position: "bottom",
-}, {
-    trigger: ".modal-footer button.btn-primary.btn-continue",
-    content: _t("Click on <b>Continue</b> to create the page."),
-    position: "bottom",
-}, {
-    trigger: "#snippet_structure .oe_snippet:eq(3) .oe_snippet_thumbnail",
-    content: _t("Drag the block and drop it in your new page."),
-    position: "bottom",
-    run: "drag_and_drop #wrap",
-}, {
-    trigger: "button[data-action=save]",
-    content: _t("Click the <b>Save</b> button."),
-    position: "bottom",
-}, {
-    trigger: ".js_publish_management .js_publish_btn",
-    content: _t("<b>That's it!</b><p>Your page is all set to go live. Click the <b>Publish</b> button to publish it on the website.</p>"),
-    position: "bottom",
-}]);
-});
-
-//==============================================================================
-
-odoo.define("website.tour.contact", function (require) {
-"use strict";
-
-var core = require("web.core");
-var tour = require("web_tour.tour");
-var _t = core._t;
-
-tour.register("contact", {
-    url: "/page/contactus",
-}, [{
-    trigger: "li#customize-menu",
-    content: _t("<b>Install a contact form</b> to improve this page."),
-    extra_trigger: "#o_contact_mail",
-    position: "bottom",
-}, {
-    trigger: "li#install_apps",
-    content: _t("<b>Install new apps</b> to get more features. Let's install the <i>'Contact form'</i> app."),
-    position: "bottom",
-}]);
-});
```

Comment:
So, should I delete the banner tour test in the same commit than the banner tour deletion ?
There is a number in each test name which is ordered, so I guess I need to adjust that number too. Should I do it in the second commit, when I rename the file in contact or should I have a 3rd commit for that ?
I read what you wrote in your previous message, just wasn't sure in which commit put the banner tour test deletion and the other tests renaming :/

Reply:
I am not sure to understand your questions...

A commit should work and be clean on its own, so yes, if you remove a JS tour in one commit, the associated test should be removed in the same commit... otherwise how would the test work if only the JS tour is removed?

I should be able to take each commit of your PR individually and Odoo should still work and not contain new dead code or missing feature.

And like I said, a renaming of file should be done without anything in the file being changed in the same commit.